src/app/pages/tabs/home/home.page.ts

```diff
@@ -189,7 +127,8 @@ export class HomePage implements OnInit {
   }
   getUser() {
     this.profileService.profileDetails().then(data => {
-      this.user = data
+      this.user = data;
+      this.isMentor = (data?.user_roles[0].title==='mentor');
```

Comment:
check for length and then check for title, else it will fail if no roles in the array.

get the value ofisMEntor from the profile service